shopfloor/actions/message.py

```diff
@@ -445,6 +445,12 @@ def packaging_not_found_in_picking(self):
             "body": _("Packaging not found in the current transfer."),
         }
 
+    def packaging_dimension_updated(self, packaging):
+        return {
+            "message_type": "success",
+            "body": _("Packaging {} dimension updated.".format(packaging.name)),
```

Comment:
this formatting is wrong
```suggestion
            "body": _("Packaging %(name)% dimension updated.", name=packaging.name),
```

Reply:
Fixed